src/popup.html

```diff
@@ -8,6 +8,10 @@
   </head>
   <body>
     <header>
+      <button id="refreshBtn" type="button" class="btn btn-outline-secondary refreshBtn">
```

Comment:
As mentioned in the previous comment:
```suggestion
      <div class="flex flex-row align-items-center justify-content-between">
        <div class="flex flex-row align-items-center">
          <img src="../images/icon48.png" width="48" height="48" class="me-3" />
          <h4>Key Retriever</h4>
        </div>
        <button id="refreshBtn" type="button" class="btn btn-outline-secondary"></button>
      </div>
```

Also we can remove that css class `refreshBtn`

Reply:
doni